### PR TITLE
Remove clues from status message

### PR DESCRIPTION
**Pull request by robertos**

### CODE REVIEW

**Comment by robertos** [comment]
Hypo feels hacky, since it doesn't know about state and reimplements game logic

**Comment by robertos** [comment]
Fixes #800

**Comment by Zamiell** [comment]
yes hypo is a short-term solution.
in the long-term, the client will do all the game logic, and then the hypo will just use that.

File: client/src/game/ui/action.ts [hunk 1]
```diff
@@ -440,42 +439,18 @@ actionFunctions.set('stackDirections', (data: ActionStackDirections) => {
...
-      // Show a yellow border around the discard pile
-      // (to reinforce that this is a "Double Discard" situation)
-      globals.elements.noDiscardBorder!.hide();
-      globals.elements.noDoubleDiscardBorder!.show();
-    } else {
-      globals.elements.noDiscardBorder!.hide();
-      globals.elements.noDoubleDiscardBorder!.hide();
-    }
+  // Update double discards
+  if (globals.clues !== MAX_CLUE_NUM
```
Comment (Zamiell) [inline client/src/game/ui/action.ts:446]: if (
  globals.clues

Comment (robertos) [reply client/src/game/ui/action.ts:446]: I believe this kind of rule can be enforced with eslint

Comment (Zamiell) [reply client/src/game/ui/action.ts:446]: which rule?

Comment (robertos) [reply client/src/game/ui/action.ts:446]: I'm checking, not sure

File: public/js/src/game/ui/sideEffects.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (Zamiell) [inline public/js/src/game/ui/sideEffects.ts]: this shouldn't be needed bcuz you have 2 exports, right?

File: client/src/game/ui/action.ts (hunk 1 above)
Comment (robertos) [reply client/src/game/ui/action.ts:446]: Hm couln't find it.

File: public/js/src/game/ui/sideEffects.ts (hunk 2 above)
Comment (robertos) [reply public/js/src/game/ui/sideEffects.ts]: Nice catch - leftover from development. Will remove

**Pull request merged**